react/QtdSelector.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+* The quantity selector component. It receives the maximous quantity that a client
```

Comment:
Remove "The" Quantity selector. Change `maximous` to `maximum`

Reply:
`the current quantity that the client already put in it and two functions, one to call when the limit is reached and another to call when the quantity is updated.` I think this shouldn't be in the class doc. Put on `handleChange` function

Reply:
Done!